lib/parlour/type_parser.rb

```diff
@@ -736,9 +736,9 @@ def parse_node_to_type(node)
           names = constant_names(target)
           known_single_element_collections = [:Array, :Set, :Range, :Enumerator, :Enumerable]
 
-          if names.length == 2 && names[0] == :T && 
+          if names.length == 2 && names[0] == :T &&
             known_single_element_collections.include?(names[1])
-            
```

Comment:
I think my editor is trimming trailing whitespace. Sorry for the noise.

Reply:
No worries - the trailing whitespace shouldn't have made it in in the first place! :P